.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -68,8 +68,8 @@ env:
   BEE_NAME: '${{ github.event.repository.name }}-${{ github.run_id }}-${{ github.run_attempt }}-dev'
   TOKEN: '${{ secrets.BROADBOT_TOKEN }}' # github token for access to kick off a job in the private repo
   E2E_ENV: 'azure_e2e.env'
-  STAGING_CHANNELS: 'C03F21QEWV7,C53JYBV9A' # C53JYBV9A channel is for #dsde-qa
-  DEV_CHANNELS: 'C03F21QEWV7' #dsp-workspaces-test-alerts
+  STAGING_CHANNELS: 'C07Q8H7F8LV,C53JYBV9A' # C53JYBV9A channel is for #dsde-qa
+  DEV_CHANNELS: 'C07Q8H7F8LV' #dsp-core-services-alerts
```

Comment:
Are we using the alerts channel for all test notifications or only failing ones?

Reply:
My general approach was to leave the alerting logic alone and just port it over to the new channel. I'm typically in favor of only alerting on failures, but this test doesn't run all that often and if we do decide the channel is too noisy then the success alerts here might be a good candidate to remove.